.travisci/codecov.sh

```diff
@@ -0,0 +1,16 @@
+#/bin/sh
+
+mvn jacoco:report
+bash <(cat $HOME/.script/codecov.bash) -c -F"unit$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
+mvn jacoco:report-integration
+bash <(cat $HOME/.script/codecov.bash) -c -F"compiler$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
+
+(cd target/it && 
+	for name in * ; do 
+		(cd "$name" && 
+			mvn jacoco:report && 
+				bash <(cat $HOME/.script/codecov.bash) -c -Funit`echo $name | sed 's/[+-]//g'`$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION || echo 'NOT POSSIBLE');
+		(cd "$name" && 
+			mvn jacoco:report-integration && 
+				bash <(cat $HOME/.script/codecov.bash) -c -Fcompiler`echo $name | sed 's/[+-]//g'`$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION || echo 'NOT POSSIBLE');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/mathieu.boretti/powerunit-extensions-matchers/pullRequest?prid=5559529)